CMakeLists.txt

```diff
@@ -459,6 +455,10 @@ if(${IREE_BUILD_COMPILER})
   add_subdirectory(build_tools/third_party/mlir-hlo EXCLUDE_FROM_ALL)
 endif()
 
+if(${IREE_ENABLE_EMITC})
+  add_subdirectory(build_tools/third_party/mlir-emitc EXCLUDE_FROM_ALL)
```

Comment:
is this needed? I don't see that path existing?

Reply:
It is, but I missed to add this file to the PR, sry.